src/firebase.ts

```diff
@@ -1,13 +1,13 @@
 import { initializeApp } from "firebase/app";
 import {
-  getAuth,
   connectAuthEmulator,
+  getAuth,
   GoogleAuthProvider,
-  EmailAuthProvider,
+  signInWithPopup,
 } from "firebase/auth";
 
 export const firebaseConfig = {
-  apiKey: "AIzaSyBVCyNLMhmnbp7CCbM3ghq_wuiEp0ZFU-c",
+  apiKey: "AIzaSyBVCyNLMhmnbp7CCbM3ghq_wuiEp0ZFU-c", // lmao
```

Comment:
I KNOW BUT https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public